packages/adapters/examples/segmentationStack/index.ts

```diff
@@ -104,7 +104,7 @@ const state = {
     toolGroup: null,
     toolGroupId: "MY_TOOL_GROUP_ID",
     viewportIds: ["CT_AXIAL"],
-    segmentationId: "LOAD_SEG_ID:" + cornerstone.utilities.uuidv4(),
+    segmentationIds: [],
```

Comment:
here we should have the segmentationId, there is one segmentation for this case and it happens to be overlapping, so don't change this to an array